### DISCUSSION TITLE
Hangisi suçlu sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#166] [Zeka Sorusu]
Ayşegül, Bahar, Mutlu , Erdem adlı dört sanıktan biri suçludur. Sorgulamalarında su önermeleri yaparlar:
Ayşegül – Mutlu suçludur. Ben suçsuzum.
Bahar- Ben suçsuzum. Suçlu bir erkektir.
Mutlu: Suçlu Ayşegüldür. Ben suçsuzum
Erdem: Ben suçsuzum. Mutlu da suçsuzdur.
Yukarıdaki söylenen cümlelerden dördü doğru, dördü yanlıştır. Buna göre suçlu kimdir?

**Comment by HabilCcabbar** [comment]
**Bahar**
Eğer Ayşegül suçlu ise:
5 doğru 3 yanlış
Eğer Mutlu suçlu ise:
5 doğru 3 yanlış
Eğer Erdem suçlu ise:
5 doğru 3 yanlış
Eğer Bahar suçlun ise:
4 doğru 4 yanlış

**Comment by begumasyaburus** [comment]
Hocam mutlu erkek mi kız mı?

**Comment by begumasyaburus** [comment]
Her kişi üzerinden denemeler yaptığımızda;
Mutlu suçlu ise cevaplar çelişiyor aynı şey ayşegül ve erdem için de geçerlidir ancak baharı suçlu kabul ettiğimizde çelişki kalmıyor ve 4 doğru 4 yanlış olarak buluyorum.
